doc/sphinx/about.rst

```diff
@@ -0,0 +1,97 @@
+.. _sec-about:
+
+*************
+About Cantera
+*************
+
+Cantera is an open source scientific computing project, that allows users to
+quickly and efficiently incorporate and execute fundamental thermodynamic,
+chemical kinetic, and species transport calculations in their work. This can be
+achieved by standalone command-line calls to core Cantera functions,
+incorporating these calls into bespoke code or software, or by using any of the
+solvers or examples distributed with the Cantera software package.
+
+Cantera was originally started by Prof. David G. Goodwin at the California
+Institute of Technology. Building on Prof. Goodwin's legacy, Cantera is licensed
+under a permissive 3-Clause BSD license, ensuring that the software will remain
```

Comment:
We could link directly to the license here.

Reply:
:+1: